spec/models/transliterated_appointment_summary_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'rails_helper'
+
+RSpec.describe TransliteratedAppointmentSummary do
+  subject(:transliterated_appointment_summary) { described_class.new(appointment_summary) }
+
+  let(:first_name) { 'Joe' }
+  let(:last_name) { 'Bloggs' }
+  let(:address_line_1) { 'A house' }
+  let(:address_line_2) { 'A street' }
+  let(:address_line_3) { 'A district' }
+  let(:town) { 'A town' }
+  let(:county) { 'A county' }
+  let(:postcode) { 'AB12 6BG' }
+  let(:country) { 'A country' }
+  let(:guider_name) { 'Mo' }
+
+  let(:appointment_summary) do
+    AppointmentSummary.new(first_name: first_name,
+                           last_name: last_name,
+                           address_line_1: address_line_1,
+                           address_line_2: address_line_2,
+                           address_line_3: address_line_3,
+                           town: town,
+                           county: county,
+                           postcode: postcode,
+                           country: country,
+                           guider_name: guider_name)
+  end
+
+  describe '.first_name' do
+    subject { transliterated_appointment_summary.first_name }
+
+    let(:first_name) { 'Noemí' }
+
+    it 'approximates non-ASCII characters' do
+      is_expected.to eq('Noemi')
+    end
```

Comment:
My 2 cents: the example doesn't really communicate the intention, so I quite like the clarity of the message (in this case)